### PR TITLE
Closed feedback session: warning message not sufficiently prominent #5271

### LINKED ISSUE #5271: Closed feedback session: warning message not sufficiently prominent
and so students may be confused as to why they cannot submit feedbacks.

image

### PR DESCRIPTION
**Pull request by YongJieYongJie**
Fixes #5271

### CODE REVIEW

**Comment by YongJieYongJie** [comment]
Rewords message and adds bold style for emphasis.
Adds modal warning prompt to confirm user reads the message.

image

**Comment by YongJieYongJie** [comment]
// currently working on passing the tests

**Comment by damithc** [comment]
The modal window title can be more specific.

**Comment by wkurniawan07** [comment]
Some tests are failing, can you get them to pass on your local machine?

File: src/main/webapp/WEB-INF/tags/shared/warningModal.tag [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<%@ tag description="Modal prompt with bootstrap warning color" %>
+<%@ taglib tagdir="/WEB-INF/tags/shared/feedbackSubmissionEdit" prefix="feedbackSubmissionEdit" %>
```
Comment (wkurniawan07) [inline src/main/webapp/WEB-INF/tags/shared/warningModal.tag]: I don't think this tag import is needed.

**Comment by YongJieYongJie** [comment]
@wkurniawan07 Ready for review!

Changes:
- Improved the modal title
- Added test method to wait for and dismiss the warning modal
  - Updated tests to use the method where necessary
- Ran godmode

image

**Comment by wkurniawan07** [comment]
There's a merge conflict here, I apologize beforehand if it's caused by me meddling with the js file when adding ESLint rules... :p

**Comment by YongJieYongJie** [comment]
@wkurniawan07 Merge conflicts resolved!

**Comment by wkurniawan07** [comment]
Looks good!

File: src/main/java/teammates/common/util/Const.java [hunk 2]
```diff
@@ -1010,7 +1010,7 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String FEEDBACK_RESPONSE_INVALID_RECIPIENT = "Trying to update recipient to an invalid recipient for question %d.";
         public static final String FEEDBACK_RESPONSE_RECIPIENT_ALREADY_EXISTS = "Error trying to update recipient for response, as another response with the same recipient already exists.";
         
-        public static final String FEEDBACK_SUBMISSIONS_NOT_OPEN = "You can view the questions and any submitted responses for this feedback session but cannot submit new responses as the session is not currently open for submission.";
+        public static final String FEEDBACK_SUBMISSIONS_NOT_OPEN = "<strong>The feedback session is currently closed.</strong> You can view the questions and any submitted responses for this feedback session but cannot submit new responses.";
```
Comment (damithc) [inline src/main/java/teammates/common/util/Const.java]: 'not open for submission' is better? Sometimes sessions are opened by accident and then put back into awaiting state (which is technically not the same as 'closed')

**Comment by damithc** [comment]
Minor change requested.

**Comment by YongJieYongJie** [comment]
@wkurniawan07 Ready for review!

As requested, I changed the warning modal message body and message title:
 - `The feedback session is currently closed.` → `The feedback session is currently not open for submissions.`
 - `Feedback Session Closed` → `Feedback Session Not Open`
image

**Comment by wkurniawan07** [comment]
Looks good :+1:

**Pull request merged**